tests/integration/records/test_indexer.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+import json
+
+from invenio_search import current_search_client as es
+
+
+def test_index_experiment_record(elastic, db, datadir, create_record):
+    data = json.loads((datadir / "1108541.json").read_text())
+    record = create_record("exp", data=data, with_indexing=True)
+    response = es.search("records-experiments")
+    expected_count = 1
+    expected_control_number = 1108541
+    assert len(response["hits"]["hits"]) == expected_count
+    assert (
+        response["hits"]["hits"][0]["_source"]["control_number"]
+        == expected_control_number
+    )
+
+
+def test_index_conference_record(elastic, db, datadir, create_record):
+    data = json.loads((datadir / "1203206.json").read_text())
+    record = create_record("con", data=data, with_indexing=True)
+
+    response = es.search("records-conferences")
+    expected_count = 1
+    expected_control_number = 1203206
+    assert len(response["hits"]["hits"]) == expected_count
+    assert (
+        response["hits"]["hits"][0]["_source"]["control_number"]
+        == expected_control_number
+    )
+
```

Comment:
IMHO it'd better to compare the whole record ``metadata`` instead of numbers (this applies in all test cases)

Reply:
fixed